main.js

```diff
@@ -92,10 +92,35 @@
     //Listener for sidebar toggle
     document.getElementById("sidebarToggle").addEventListener("click", toggleSidebar);
 
-    // Display a notification to the user
-    function displayNotification(messageText) {
-      // TODO: Actually make the notification instead of logging
-      console.log(messageText);
+    // Display a notification to the user.
+    // Style is optional, can be "error", "warning", or "success"
+    function displayNotification(messageText, style) {
+        var notificationArea = document.getElementById("notifications");
+        var box = document.createElement("div");
+        box.className = "notification";
+
+        if (style != undefined) {
+            box.className += " " + style;
+        }
+
+        var closeButton = document.createElement("a");
```

Comment:
If it's just going to be an empty link, should it just be a `button` instead? With an onclick handler? With take some effort to unstyle the default button.